lib/graphql/field/resolve.rb

```diff
@@ -51,7 +61,11 @@ def initialize(field)
         end
 
         def call(obj, args, ctx)
-          obj.public_send(@field.name)
+          if method_with_arguments?(@field.name, obj, args)
+            obj.public_send(@field.name, args)
```

Comment:
@cjoudrey I updated the pull request to send context beside arguments.

This is an example code that accepts arguments and context:
```rb
def todo_add(args, ctx) # this is a method and accepting arguments
    todo = Todo.new(args[:text], ctx[:context_value])
    @todos << todo
    todo
end
```